### PR TITLE
feat(lb): simplify argument passing for certificate creation

### PR DESCRIPTION
**Pull request by remyleone**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request.
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Release note for [CHANGELOG](https://github.com/scaleway/scaleway-cli/blob/v2/CHANGELOG.md):

```release-note

```

### CODE REVIEW

File: internal/core/testing.go [hunk 1]
```diff
@@ -403,9 +403,16 @@ func Test(config *TestConfig) func(t *testing.T) {
 		// Run config.Cmd
 		var result interface{}
 		var exitCode int
-		args := config.Args
+		var renderedArgs []string
+		rawArgs := config.Args
 		if config.Cmd != "" {
-			args = cmdToArgs(meta, config.Cmd)
+			renderedArgs = cmdToArgs(meta, config.Cmd)
```
Comment (remyleone) [inline internal/core/testing.go:409]: We should simplify this code to avoid using cmdToArgs

File: internal/namespaces/lb/v1/helper_test.go [hunk 2]
```diff
@@ -14,3 +16,37 @@ func createLB() core.BeforeFunc {
...
+
+func setUpCertificateLocally() core.BeforeFunc {
+	return func(ctx *core.BeforeFuncCtx) error {
+		homeDir := ctx.OverrideEnv["HOME"]
+		privateKeyPath := path.Join(homeDir, "private.key")
+		privateCSRPath := path.Join(homeDir, "private.csr")
+		certificatePath := path.Join(homeDir, "certificate.crt")
+
+		// Generate private key and Certificate Signing Request (CSR)
+		cmd := []string{
```
Comment (remyleone) [inline internal/namespaces/lb/v1/helper_test.go:28]: This kind of openssl command should be in the long section in the help to be copy and pasted easily

File: internal/core/testing.go [hunk 3]
```diff
@@ -403,9 +403,15 @@ func Test(config *TestConfig) func(t *testing.T) {
 		// Run config.Cmd
 		var result interface{}
 		var exitCode int
-		args := config.Args
+		var renderedArgs []string
```
Comment (kindermoumoute) [inline internal/core/testing.go:406]:
```suggestion
		renderedArgs :=  []string(nil)
```

**Review by kindermoumoute — APPROVED** [review]
LGTM

**Pull request merged**